src/client/chrome/scripts/background.js

```diff
@@ -1,16 +1,20 @@
 //listens when a tab is opened, page is visited
 chrome.tabs.onUpdated.addListener(function (tabId, changeInfo, tab) {
   var domain = tab.url.replace('http://','').replace('https://','').split(/[/?#]/)[0];
+
   if(changeInfo.status == 'complete' && tab.title){
       if(tab.url != 'chrome://newtab/'){
+
         fetch('http://127.0.0.1:8000/newpage/', {
           headers: {
             'Accept': 'application/json',
             'Content-Type': 'application/json'
           },
           method: "POST",
-          body: JSON.stringify({"tab":tab.id, "title":tab.title, "domain":domain, "url":tab.url})
+          body: JSON.stringify({"tab":tab.id, "title":tab.title, "domain":domain, "url":tab.url, "favIconUrl":tab.favIconUrl, "previousTabId": 0})
```

Comment:
This isn't the right post. You should pull develop into this branch and make sure the post is right